theme/static_src/src/djangonaut-space.css

```diff
@@ -79,14 +79,14 @@
 ----------------------------------------------------------------------*/
 
 .landing {
-    max-width: 100vw;
+    /*max-width: 100vw;*/
     min-height: 100vh;
     min-height: 100vh;
     margin: 0 auto;
     padding: 0 15px;
     display: flex;
     flex-direction: column;
-    justify-content: center;
+    /*justify-content: center;*/
```

Comment:
Is there a reason why these are commented out rather than deleted?

Reply:
I wasn't sure if it was the right file to update.

Reply:
Gotcha. I think we need to decide if it is and then either remove or add the code. Commented out code can be confusing to people, especially when debugging a problem. If I do need to commit commented or code, I like to leave an explicit reason why it's commented out and when it should be uncommented or removed.

Reply:
Ok, in this case, I will remove `justify-content: center` because I use the breakpoints in the body class. I don't find any other reference for this class.